overrides/partials/footer.html

```diff
@@ -73,8 +73,9 @@
       </div>
 
       <div class="footer-links md-footer-copyright">
+        <a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademarks</a> |
         <a href="https://policies.google.com/privacy">Privacy Policy</a> |
-        <a href="/development/about-analytics-cookies/">Cookie Usage</a> |
+        <a href="/docs/about-analytics-cookies/">Cookie Usage</a> |
```

Comment:
we should also update this link on L113 below (in the cooke banner)

Reply:
Nice catch 👍